src/components/views/DownloadProgressModal.vue

```diff
@@ -23,28 +25,45 @@ function closeModal() {
             <div class='modal-content'>
                 <div class='notification is-info'>
 
-                    <h3 v-if="$store.getters['download/currentDownload'].downloadProgress < 100" class='title'>
+                    <h3 v-if="DownloadUtils.statusIsDownloadOrExtract($store.getters['download/currentDownload'].status)" class='title'>
                         Downloading {{$store.getters['download/currentDownload'].modName}}
                     </h3>
-                    <h3 v-else class='title'>
+                    <h3 v-else-if="$store.getters['download/currentDownload'].status === DownloadStatusEnum.INSTALLING" class='title'>
                         Installing {{$store.getters['download/currentDownload'].modName}}
                     </h3>
 
-                    <p>
+
+                    <p v-if="$store.getters['download/currentDownload'].status === DownloadStatusEnum.DOWNLOADING">
+                        <font-awesome-icon icon="fa-regular fa-download" />
```

Comment:
On r2mm the console is peppered with errors:

```
Unknown custom element: <font-awesome-icon> - did you register the component correctly? For recursive components, make sure to provide the "name" option.

found in

---> <DownloadProgressModal> at src/components/views/DownloadProgressModal.vue
```

Not sure if it's this line or one of the other icons.

Reply:
Ooops, didn't test properly after adding the icons 🤦‍♂️ Fixed now.